modules/dataquery/ajax/saveQuery.php

```diff
@@ -43,6 +63,14 @@
 $cond   = $_REQUEST['Filters'];
 $baseDocument['Conditions'] = $cond;
 $baseDocument['Fields']     = $fields;
-$cdb = CouchDB::singleton();
-print $cdb->postDoc($baseDocument);
+
+if ($_REQUEST['Override'] === "true") {
```

Comment:
I don't think this variable name is very clear.

What's it overriding? Maybe something like "OverwriteQuery" would be a better name?